client/my-sites/email/email-management/home/utils.js

```diff
@@ -167,3 +169,18 @@ export function resolveEmailPlanStatus( domain, emailAccount, isLoadingEmails )
 
 	return activeStatus;
 }
+
+/**
+ * Tracks an event for the key 'calypso_email_app_launch'. It will not track events from development mode.
+ *
+ * @param {app, context, provider} - app/context/provider should be string and must be provided.
+ */
+export function recordEmailActionTrackEvent( { app, context, provider } ) {
```

Comment:
I think we should call this `recordEmailActionTracksEvent` with the `s` -- we call the solution `Tracks` internally.

Also, I am wondering whether it makes sense to call this something closer to its purpose, which is specifically to track app launches. Might it be worth calling it `recordEmailAppLaunchEvent`?

Reply:
I've definitely called it `recordEmailAppLaunchEvent`